requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
-pytest==2.7.2
+pytest==2.9.2
 pytest-cov==2.1.0
+git+git://github.com/tilery/utilery.git
```

Comment:
Having only installed the requirements.txt and requirements-dev.txt dependencies without utilery, the error I received was: 

```
_______________________________________ test_import_by_path_should_return_module_from_path ________________________________________

    def test_import_by_path_should_return_module_from_path():
>       assert import_by_path('utilery.utils.import_by_path') == import_by_path

tests/test_utils.py:5:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
aiovectortiler/utils.py:14: in import_by_path
    module = import_module(module_path)
../../.virtualenvs/aiovectortiler/lib/python3.5/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:986: in _gcd_import
    ???
<frozen importlib._bootstrap>:969: in _find_and_load
    ???
<frozen importlib._bootstrap>:944: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:222: in _call_with_frames_removed
    ???
<frozen importlib._bootstrap>:986: in _gcd_import
    ???
<frozen importlib._bootstrap>:969: in _find_and_load
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

name = 'utilery', import_ = <function _gcd_import at 0x10e1a0048>

>   ???
E   ImportError: No module named 'utilery'

<frozen importlib._bootstrap>:956: ImportError
```

Since utilery isn't on PyPI, this was the cleanest way I could come up with to install it with pip!